mk/spksrc.service.installer.functions

```diff
@@ -98,7 +98,7 @@ reload_inst_variables ()
         for _line in $(cat "${INST_VARIABLES}"); do
             _key="$(echo ${_line} | awk -F'=' '{print $1}')"
             _value="$(echo ${_line} | awk -F'=' '{print $2}')"
-            declare "${_key}=${_value}"
```

Comment:
It seems that `declare ` on its own and used within a function wouldn't be overriding the parent context, making this call actually not updating the parent context. Unless I missed something ?